src/features/users/AccountDropdown.tsx

```diff
@@ -14,25 +14,32 @@
  * limitations under the License.
  */
 
-import React, { useCallback } from 'react';
-import { useNavigate } from 'react-router-dom';
-
+import React, { useCallback, useState, useEffect } from 'react';
 import { useAppSelector, useAppDispatch } from 'app/hooks';
 import { selectUsers, logoutUser } from './usersSlice';
 import { Popover, Dropdown } from 'components';
 
 export function AccountDropdown() {
   const { username } = useAppSelector(selectUsers);
+  const [opened, setOpened] = useState(false);
 
-  const navigate = useNavigate();
   const dispatch = useAppDispatch();
   const logout = useCallback(() => {
     dispatch(logoutUser());
-    navigate('/login');
-  }, [dispatch, navigate]);
+  }, [dispatch]);
+
+  useEffect(() => {
+    const handleResize = () => {
+      if (window.innerWidth < 1024) {
```

Comment:
Same here.

Reply:
The magic number is very difficult to maintain later.
https://en.wikipedia.org/wiki/Magic_number_(programming)